iOS/DuckDuckGo/AppServices/WideEventService.swift

```diff
@@ -21,13 +21,15 @@ import Foundation
 import BrowserServicesKit
 import PixelKit
 import Subscription
+import VPN
 
 final class WideEventService {
     private let wideEvent: WideEventManaging
     private let featureFlagger: FeatureFlagger
     private let subscriptionBridge: SubscriptionAuthV1toV2Bridge
     private let activationTimeoutInterval: TimeInterval = .hours(4)
     private let restoreTimeoutInterval: TimeInterval = .minutes(15)
+    private let vpnConnectionTimeoutInterval: TimeInterval = .minutes(15)
```

Comment:
Eventually I think these timeout intervals should be defined on the events themselves, which I have a WIP branch for as part of cleaning up how we handle expired pixels - expecting to have that up early next week.

Reply:
My only thought here is 15 minutes is a lot, and introduces a higher risk the user might kill the app and turn the pixel into an abandoned pixel which gets reported with a different reason.

I think it might make sense to reduce this to something like 2 minutes - which should be plenty for the VPN connection.